apps/lockfile-explorer-server/package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This project is missing a `LICENSE` file.

Reply:
added